extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/DataSources.java

```diff
@@ -206,45 +205,6 @@ public AgroalDataSource doCreateDataSource(String dataSourceName, boolean failIf
 
         String jdbcUrl = dataSourceJdbcRuntimeConfig.url().get();
 
-        if (dataSourceJdbcBuildTimeConfig.tracing()) {
```

Comment:
I think you also need to remove these constants on the top of the class:
```java
    public static final String TRACING_DRIVER_CLASSNAME = "io.opentracing.contrib.jdbc.TracingDriver";
    private static final String JDBC_URL_PREFIX = "jdbc:";
    private static final String JDBC_TRACING_URL_PREFIX = "jdbc:tracing:";
```

Reply:
PR Updated! :)